packages/combobox/src/utils.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { KEYS } from '@zendeskgarden/container-utilities';
+import { useCombobox as useDownshift } from 'downshift';
+
+/**
+ * Convert Downshift types to Garden.
+ *
+ * @param downshiftType A Downshift state change type.
+ *
+ * @returns A preferred Garden type.
+ */
+export const toType = (downshiftType: string) => {
+  /* istanbul ignore next */
+  switch (downshiftType) {
+    case useDownshift.stateChangeTypes.FunctionCloseMenu:
```

Comment:
Wondering if this might be simpler as an object, and if a value is `undefined`, just return the default case. Should be fewer lines of code, at least, since you can skip the empty `case`s.

```
const GardenTypeMap = {
  [useDownshift.stateChangeTypes.FunctionToggleMenu]: "fn:setExpansion",
  ...
}

export const toType = (downshiftType: string) => {
  return GardenTypes[downshiftType] || downshiftType 
}
```